core/modules/measure.cc

```diff
@@ -38,50 +38,48 @@ void Measure::ProcessBatch(bess::PacketBatch *batch) {
     start_time_ = get_time();
   }
 
-  if (static_cast<int>(HISTO_TIME_TO_SEC(time - start_time_)) < warmup_) {
+  if (static_cast<int>(HISTO_TIME_TO_SEC(time - start_time_)) >= warmup_) {
     RunNextModule(batch);
-    return;
-  }
-
-  pkt_cnt_ += batch->cnt();
+    pkt_cnt_ += batch->cnt();
 
-  for (int i = 0; i < batch->cnt(); i++) {
-    uint64_t pkt_time;
-    if (get_measure_packet(batch->pkts()[i], &pkt_time)) {
-      uint64_t diff;
+    for (int i = 0; i < batch->cnt(); i++) {
+      uint64_t pkt_time;
+      if (get_measure_packet(batch->pkts()[i], &pkt_time)) {
+        uint64_t diff;
 
-      if (time >= pkt_time) {
-        diff = time - pkt_time;
-      } else {
-        continue;
-      }
+        if (time >= pkt_time) {
+          diff = time - pkt_time;
+        } else {
+          continue;
+        }
 
-      bytes_cnt_ += batch->pkts()[i]->total_len();
-      total_latency_ += diff;
+        bytes_cnt_ += batch->pkts()[i]->total_len();
+        total_latency_ += diff;
 
-      record_latency(&hist_, diff);
+        record_latency(&hist_, diff);
+      }
     }
+    RunNextModule(batch);
```

Comment:
Oops, it was a merge conflict, sorry I did not build before submitting (serves me right). Let me fix it and update it shortly.